services/classic/management/virtualmachine/entities.go

```diff
@@ -71,7 +71,7 @@ type DeploymentResponse struct {
 	ExtendedProperties     []ExtendedProperty `xml:">ExtendedProperty"`
 	PersistentVMDowntime   PersistentVMDowntime
 	VirtualIPs             []VirtualIP `xml:">VirtualIP"`
-	ExtensionConfiguration string      // cloud service extensions not fully implemented
+	ExtensionConfiguration ExtensionConfiguration
```

Comment:
This is a breaking change so we'll have to release this as part of v25 at the end of this month.

Reply:
@jhendrixMSFT . Sounds good if we need to wait for v25, even though the property would always be null previously.